pkg/tanka/export.go

```diff
@@ -43,7 +43,7 @@ type ExportEnvOpts struct {
 	Parallelism int
 }
 
-func ExportEnvironments(paths []string, to string, opts *ExportEnvOpts) error {
+func ExportEnvironments(paths map[string]string, to string, opts *ExportEnvOpts) error {
```

Comment:
It is quite unobvious what this `map[string]string` should hold. From looking at the code, it appears to be a name -> entrypoint mapping.

Wdyt about passing a `[]Environment` instead, where the elements are expected to be sparse environments (missing `data` key)?